spec/controller/metrics_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe MetricsController, type: :controller do
```

Comment:
Can you make this a requests spec instead?  I believe controller specs are deprecated.

Reply:
Will do, thx

Reply:
Done